plugins/nodemon/src/tasks/nodemon.ts

```diff
@@ -0,0 +1,72 @@
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { hookFork, styles } from '@dotcom-tool-kit/logger'
+import { Task } from '@dotcom-tool-kit/types'
+import { NodemonOptions, NodemonSchema } from '@dotcom-tool-kit/types/lib/schema/nodemon'
+import { VaultEnvVars } from '@dotcom-tool-kit/vault'
+import getPort from 'get-port'
+import nodemon from 'nodemon'
+import { Readable } from 'stream'
+
+export default class Nodemon extends Task<typeof NodemonSchema> {
+  static description = ''
+
+  static defaultOptions: NodemonOptions = {
+    entry: './server/app.js'
+  }
```

Comment:
**question:** something else I suddenly thought of is that [in dotcom-debug-user](https://github.com/Financial-Times/dotcom-debug-user/blob/main/.toolkitrc.yml#L34), we're having to specify `jsx` as a file extension quite often. Is adding in `jsx` by default something that Platforms would consider? I don't feel like it would cause issues for projects which don't use JSX for templating.

[Nodemon makes this configurable with the `ext` option](https://github.com/remy/nodemon/#specifying-extension-watch-list), so I _think_ this could be adjusted to something like this (which adds `jsx` to Nodemon's defaults and ditches some of the extensions we don't use like CoffeeScript):

```suggestion
  static defaultOptions: NodemonOptions = {
    entry: './server/app.js',
    ext: ['cjs', 'js', 'json', 'jsx', 'mjs']
  }
```

(would need to update the types as well I think)

Reply:
This is probably something that should be done with a [`nodemon.json` config file in your project](https://github.com/remy/nodemon#config-files) rather than having Toolkit handle it specifically. I don't think we should be overriding any configuration defaults within Tool Kit if we can help it, although `.jsx` support would be a pretty common use case. What do you think, @apaleslimghost?

Reply:
yep, the general philosophy for Tool Kit is not to provide another method of configuration if the underlying tool has its own way of doing that (e.g. we don't allow specifying Webpack config options in `.toolkitrc.yml`). honestly? i'd open a PR on Nodemon to add `.jsx` to the default extensions 😅

Reply:
I was tempted to go and do this, but then [this discussion](https://github.com/remy/nodemon/issues/1954) about adding `cjs` didn't really go anywhere and the [matching PR](https://github.com/remy/nodemon/pull/1957) hasn't been touched since November so I might just leave it. Fine to continue configuring ourselves as we have to do this for other plugins anyway 👍